exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/WebUserConnection.java

```diff
@@ -106,7 +110,30 @@ public void sendData(RpcOutcomeListener<Ack> listener, QueryWritableBatch result
         // TODO:  Clean:  DRILL-2933:  That load(...) no longer throws
         // SchemaChangeException, so check/clean catch clause below.
         for (int i = 0; i < loader.getSchema().getFieldCount(); ++i) {
-          columns.add(loader.getSchema().getColumn(i).getName());
+            //DRILL-6847:  This section adds query metadata to the REST results
```

Comment:
@cgivre sorry for being picky but it seems indent in for loop and if clauses does not qualify to the accepted one in Drill project.

Reply:
No worries... Fixed.